include/z64player.h

```diff
@@ -383,6 +383,8 @@ typedef struct {
     /* 0x10 */ Vec3f base;
 } WeaponInfo; // size = 0x1C
 
+#define PLAYER_GI_SEG_MIN 0x2880 // max size of a place title card file
```

Comment:
maybe this could go in config_game.h? unless the value should always be 0x2880

Reply:
i dont see people touching this value if the game auto updates it

Reply:
if users shouldn't edit this then this is fine yeah

Reply:
This is more of a placeholder for now until I can think of a better way to handle it. The get item buffer is also used for the place name textures, which ironically are actually larger than any of the vanilla get item objects.